airflow/providers/google/cloud/hooks/vertex_ai/auto_ml.py

```diff
@@ -185,6 +186,11 @@ def get_auto_ml_image_training_job(
             model_encryption_spec_key_name=model_encryption_spec_key_name,
         )
 
+    @deprecated(
+        planned_removal_date="December 15, 2025",
```

Comment:
Should it be here June 15, 2025?

Reply:
The idea was June 15 2025(deprecation date) + 6 month.
 Is it incorrect suggestion?

Reply:
All deprecation dates got updated to the dates suggested in google documentation, thank you for highlighting this detail!